scripts/sync.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "fs";
+
+/**
+ * This script synchronizes the in-repo ecosystem schema JSON to the latest
+ * version.
+ */
+
+async function updateSchema() {
+    console.log("Updating games.json...")
```

Comment:
Nit: we can probably drop the console.log but not overly bothered

Reply:
This is only shown when you run a CLI command in the project so that you can keep track of what's going on, as it was hard to tell without any status indicators.